pkg/metrics/statsd/statsd.go

```diff
@@ -103,13 +108,42 @@ func (s *statsd) run() {
 	}
 }
 
+func (s *statsd) getConnectionDetails() (protocol string, connectionstring string, err error) {
+	// allow the socket connection to be overriden via ENV Variable
+	o, isset := os.LookupEnv(STATSD_SOCKET_ENV)
```

Comment:
That would be great to have more expressive names

Reply:
“There are only two hard things in Computer Science: cache invalidation and naming things.” . In the past when I got comments on my variables  naming it was to about names being to long and too python-style.
Any general guidelines you could point me to ? Any concrete suggestions here ?

Reply:
I don't know what _too python-style_ is, but I disagree with whoever said that. Names have to be descriptive. I'd rather have a name that is a little long but I know by just looking at it what the variable holds, rather than having to go back and forth to figure it out  

here I'd say maybe something like `socketEnv` could do, it's not perfect but at least we know what the variable holds. WDYT?

Reply:
Updated the variable names accordingly